docs/contributing/index.rst

```diff
@@ -30,6 +30,16 @@ You can use `make` to install all tsuru dependencies and run tests. It will also
 
     $ make
 
+Please ensure yourself that MongoDB and Redis are started before running the tests (if you are seeing some failing tests the most likely reason is that these services are not started).
```

Comment:
I think we can drop the "yourself" and improve the text to detail the error, something like:

```
Please ensure that MongoDB and Redis are started before running test suite. If you see some tests failures with messages like "dial tcp 127.0.0.1:6379: connection refused" and "no reachable server", the most likely reason is that these services are not running.
```

Thank you!